.gitignore

```diff
@@ -1,4 +1,5 @@
 __pycache__
+*.mo
```

Comment:
@just-paja Thats right. That file was there by mistake. Now the `.mo` file is generated during CicrceCI compilation and should be included in the sdist package.